tests/rados/pool_tests.py

```diff
@@ -1439,6 +1408,132 @@ def run(ceph_cluster, **kw):
         log.info(" Verification of pg_min_num complete")
         return 0
 
+    if config.get("Verify_ec_profile"):
+        log.info(
+            "\nTest to verify the behavior of EC profiles when attempting to modify and delete them."
+            f" \nTests running on RHCS version : {rhbuild}\n"
+        )
+        profile_configs = config["Verify_ec_profile"]
+        test_pass = True
+
+        try:
+            log.debug("Creating test pool and profile to verify functionality")
+            if not rados_obj.create_erasure_pool(**profile_configs):
+                log.error("Failed to create the EC Pool")
+                return 1
+
+            rados_obj.bench_write(**profile_configs)
+            rados_obj.bench_read(**profile_configs)
+            log.info("Created the EC Pool and finished writing data into the pool.")
+
+            old_value = profile_configs["k"]
+            profile_configs["k"] += 1  # Increment the 'k' value
+            profile_configs["create_ecpool"] = False
+
+            # Modify profile without --force flag
+            log.info("Attempt 1: Modifying the profile without the --force flag")
+            try:
+                rados_obj.create_erasure_pool(**profile_configs)
+            except Exception as err:
+                log.info(
+                    f"Expected exception. Cannot modify the profile without --force flag.\n"
+                    f"Expected Error: {err}"
+                )
+
+            out = rados_obj.get_ec_profile_detail(
+                profile=profile_configs["profile_name"]
+            )
+
+            if out and int(out["k"]) != old_value:
+                log.error(
+                    f"Profile updated. Error! Current Value : {out['k']} , Expected : {old_value}"
+                )
+                test_pass = False
+
+            # Modify profile with --force flag
+            log.info("Attempt 2: Modifying the profile with the --force flag")
+            try:
+                rados_obj.create_erasure_pool(**profile_configs, force=True)
+            except Exception as err:
+                log.warning(f"Modification failed with --force flag. Exception: {err}")
```

Comment:
If it is expected that the profile gets modified, why are we not stopping the test?